submissions/OwoeyeGbenga/landing_page/css/style.css

```diff
@@ -0,0 +1,389 @@
+html {
```

Comment:
Well done. Please rename your folder to match the exact task you are submitting in this pull request

Reply:
I named the folder landing_page. It contains my homepage file and links to  my signup and login page